build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.3'
+        classpath 'com.android.tools.build:gradle:7.0.4'
```

Comment:
Please revert this? we do the upgrade the version internally to make sure that  the latest version won't cause any issues for internal pipelines +. it is not related to the issue mentioned

Reply:
Done.